libraries/pom.xml

```diff
@@ -39,12 +39,31 @@
                 <artifactId>maven-surefire-plugin</artifactId>
                 <version>${maven-surefire-plugin.version}</version>
                 <configuration>
+                    <testFailureIgnore>true</testFailureIgnore>
+                    <includes>
+                        <include>**/*Test.java</include>
+                        <include>**/GithubUserProfilePayload.java</include>
```

Comment:
`GithubUserProfilePayload.java` is indeed a test case, but the Test file name should be in accord with the story name: `github_user_profile_payload.story`. I think it's forced by Serenity.
I did a workaround: renamed the story file name to `github_user_profile_payload_live_test.story`, so that the test can detect the relevant story file.